docs/book/src/reference/admission-webhook.md

```diff
@@ -0,0 +1,38 @@
+# Admission Webhook
+
+Admission webhooks are HTTP callbacks that receive admission requests, process them and return admission responses.
+There are two types of admission webhooks: mutating admission webhook and validating admission webhook.
```

Comment:
May be use list/bullets to enumerate the two webhooks ? 

Kubernetes provides following types of admission webhooks:
  - Mutating Admission Webhook: ....desc.....
  - Validating Admission Webhook: .....desc.....